src/Records/Command.php

```diff
@@ -48,9 +48,11 @@ public function __construct(
         public int $cache_events,
         public int $hydrated_models,
         public int $peak_memory_usage,
+        public string $exception_preview,
     ) {
         $this->class = Str::text($this->class);
         $this->name = Str::tinyText($this->name);
         $this->command = Str::text($this->command);
+        $this->exception_preview = Str::text($this->exception_preview);
```

Comment:
Think these should be tiny text? From the ingest:

<img width="544" alt="Screenshot 2025-03-26 at 09 49 12" src="https://github.com/user-attachments/assets/712c1ee6-a625-4ea4-96fc-db2ffd941fe5" />

```suggestion
        $this->exception_preview = Str::tinyText($this->exception_preview);
```